src/react-native-app/android.Dockerfile

```
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# https://github.com/react-native-community/docker-android
+# Choosing a tag where the Android build tools match what we have in android/build.gradle to avoid the
+# container having to download them
+FROM reactnativecommunity/react-native-android:v13.2.1 AS builder
+
+WORKDIR /reactnativesrc/
+COPY . .
+
+RUN npm install
+WORKDIR android/
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)